tests/python/unittest/test_numpy_op.py

```diff
@@ -1326,7 +1325,7 @@ def hybrid_forward(self, F, a):
 
 @with_seed()
 @use_np
-@unittest.skip("NumpyBooleanAssignForwardCPU broken: https://github.com/apache/incubator-mxnet/issues/17990")
+@pytest.mark.skip(reason="NumpyBooleanAssignForwardCPU broken: https://github.com/apache/incubator-mxnet/issues/17990")
```

Comment:
This should haven been fixed by https://github.com/apache/incubator-mxnet/pull/18158 so let's remove the skip